devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -68,11 +68,11 @@ metadata {
 
 		// Yale siren only
 		preferences {
-			input name: "alarmLength", type: "number", title: "Alarm length (1-10 min)", range: "1..10"
+			input name: "alarmLength", type: "number", title: "Alarm length", range: "1..10"
```

Comment:
`Length` should be capitalized. Also maybe change it to `Duration`? Finally, might want to add a description and include that the units is minutes.

Reply:
I think these have already been sent of for translation.

Reply:
The changes in this PR have been sent off or the original strings?

Reply:
the changes in this PR